test/nodes/test_audio.py

```diff
@@ -5,68 +5,120 @@
 
 try:
     import soundfile as sf
+    import ffmpeg
 
     soundfile_not_found = False
 except:
     soundfile_not_found = True
 
+from transformers import WhisperProcessor, WhisperForConditionalGeneration
+
 from haystack.schema import Span, Answer, SpeechAnswer, Document, SpeechDocument
 from haystack.nodes.audio import AnswerToSpeech, DocumentToSpeech
 from haystack.nodes.audio._text_to_speech import TextToSpeech
 
 from ..conftest import SAMPLES_PATH
 
+WHISPER_MODEL = "openai/whisper-medium"
```

Comment:
nit: since we're only using it at one place and we are also planning to move audio node out of haystack, I wouldn't mind if we hardcode.